### DISCUSSION TITLE
Using modified Apsim plant models in publication work

### DISCUSSION THREAD

**Discussion by LenLon** [APSIMInitiative/ApsimX#8570] [Announcements]
### What is your question?

Hello,

I am currently working on heat stress as a possible crop varietal adaptation under climate change. Considering that heat stress is not implemented in ApsimX's crop models (yet), many users (it seems) make use of the modularity of the PMF in ApsimX to introduce heat stress effects in ApsimX's plant models, either based on other publications or previously implemented approaches in Apsim classic. 

Now that replacements are read-only by default to discourage the implementation of non-validated models for "routine application tasks", I was wondering what the stance is on using these models for scientific impact studies which might end up being peer-reviewed and published? Does the end-user license cover this? I assume it should be made clear in the paper that the plant model has been modified and how, but can you still call it an Apsim study? What would the correct approaches for validation be, considering we are often simulating under projected or counterfactual climates? 

Though this question only indirectly pertains to ApsimX development, I thought discussion around this could be stimulating for everyone. I'm happy to take this to email if better suited.

**Comment by hol353** [comment]
These are great questions. Moving to discussions section (which we're migrating to for these sorts of questions).

**Comment by hol353** [comment]
The end-user license says that if you modify a model then it has to be submitted back to the APSIM Initiative (AI) for peer-review and potential inclusion into APSIM. If this isn't done then it is a breach of the end-user licence. Once submitted, the AI will look for evidence that the change works as expected (e.g. comparisons with observed field data, perhaps comparisons with 'expert' opinion or expected behaviour in some cases). They might offer some suggestions for improvement and if accepted then change will be include and made available in the next release of APSIM so that others can benefit. The process is a bit like a journal review process.

When publishing a paper that uses a modified model then it should be made clear what change was made. As an example, if the wheat model was modified and it hasn't gone through AI approval then it shouldn't be called the APSIM wheat model.

> What would the correct approaches for validation be, considering we are often simulating under projected or counterfactual climates?

You only need a validation if a new model is created or an existing model is improved. If you are running APSIM under future climates then that is a scenario analysis and doesn't really require a validation. Of course you need to convince yourself that the model is doing sensible things i.e. the model is behaving as expected under higher CO2 or temperatures and you should include this in the journal paper.

**Comment by hol353** [comment]
Tagging some teams to this interesting discussion starter as the new discussion forum on GitHub may be missed by lots of people.

@APSIMInitiative/steering-committee 
@APSIMInitiative/reference-panel 
@APSIMInitiative/developers
